apps/fabric-website/src/components/IconGrid/IconGrid.tsx

```diff
@@ -56,36 +58,84 @@ export class IconGrid extends React.Component<IIconGridProps, IIconGridState> {
     );
   }
 
+  public async componentDidMount() {
+    // Load the icons async after mount to avoid them being bundled and parsed with the page,
+    // due to their large bundle size.
+    switch (this.props.iconType) {
+      case 'font':
+        // The explicit json-loader! import helps prevent this from being processed by TS
+        // (we want only webpack to handle it)
+        const fontIcons = (await import('json-loader!@uifabric/icons/lib/data/AllIconNames.json')) as IIconInfo[];
+        // There's also a metadata entry in this file which doesn't have a name
+        this.setState({ icons: fontIcons.filter(icon => !!icon.name) });
```

Comment:
Filtering this out allows us to avoid a duplicate key warning on the `li` elements below without using the index as part of the key (it's generally not recommended to use indices as keys)